### DISCUSSION TITLE
Lycoris, Locon, LoHa

### DISCUSSION THREAD

**Discussion by KaoruAsoi** [etherealxx/etherportal-webui-colab#6] [Ideas]
I honestly don't know what Lycoris, Loha etc are, but can you add this? There are some models that use Lycoris instead of Lora and I am quite confused using the Lycoris model because Lycoris is not in CivitAi tab, Thanks mate
![Screenshot 2023-03-22 203416](https://user-images.githubusercontent.com/120241906/226921052-fa4bb35f-effb-410d-ad64-fc7775e6e52e.png)

**Comment by etherealxx** [comment]
Hello @KaoruAsoi 👋 yes there's this thing called Lycoris, which is a blanket term of LoCon and LoHa. Both identifies as 'LoCon' on civitai's json file (even though the website says the type is 'lycoris'). The way it activated is by using the additional networks extension. (which means using #addnetlora in batchlinks will be fine for that.)
However, i think i'll make a new hashtag for it, it will make a subfolder inside the #addnetlora path, so that it is differentiated, but still works as expected.

Meanwhile i'll ask Camenduru about the civitai extension, since me and him are the one maintaining it now, kind of.
